src/components/Recorder/index.test.js

```diff
@@ -1,79 +1,178 @@
-/**
- * @jest-environment jsdom
- */
-import React from 'react';
-import ReactDOM from 'react-dom';
-import renderer from 'react-test-renderer';
-import { render, screen, logRoles, prettyDOM } from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
-import '@testing-library/jest-dom/extend-expect'
```

Comment:
Thank you for cleaning all of this up. There were a lot of boilerplate imports here and I wasn't sure which I could delete!